splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/CachedOperation.java

```diff
@@ -71,8 +71,8 @@ public void init(SpliceOperationContext context) throws StandardException {
             super.init(context);
             source.init(context);
             if (!populated) {
-                populateCache();
                 populated = true;
+                populateCache();
             }
```

Comment:
Can you add a comment for ordering changes, so that no one may accidentally reorder it?

Reply:
Sure, will do, thanks!